src/_guides/testing.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: "Dart Testing"
+description: "How to test Flutter, Web, and VM Applications."
+---
+
+Software testing, an important part of app development, helps verify that
+your app is working correctly before releasing it to your users.
+This Dart testing guide outlines several types of testing, and points
+you to where you can learn how to test your
+[mobile](https://flutter.io/), [web](https://webdev.dartlang.org/),
+and [server-side apps and scripts](/dart-vm).
+
+<aside class="alert alert-info" markdown="1">
+**Terminology: widget vs. component**<br>
+Flutter, an SDK for building apps for iOS and Android, defines its
+GUI elements as _widgets_. AngularDart, a web app framework,
+defines its GUI elements as _components_.
+This doc uses **component** (except when explicitly discussing Flutter),
+but both terms refer to the same concept.
+</aside>
+
+## Testing types
```

Comment:
Maybe the section title should be changed to match the wording change in the prose. Possibly:
> ## Kinds of testing